spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -296,79 +296,117 @@
       end
 
       context 'when an enrollment passes' do
-        before(:each) do
-          stub_request_passed_proofing_results
-        end
+        let!(:response) { stub_request_passed_proofing_results }
 
         it_behaves_like('enrollment with a status update', passed: true, status: 'passed')
 
         it 'logs details about the success' do
           job.perform(Time.zone.now)
 
+          response_as_json = JSON.parse(response[:body])
           expect(job_analytics).to have_logged_event(
             'GetUspsProofingResultsJob: Enrollment status updated',
-            fraud_suspected: false,
             reason: 'Successful status update',
+            fraud_suspected: response_as_json['fraudSuspected'],
+            primary_id_type: response_as_json['primaryIdType'],
+            secondary_id_type: response_as_json['secondaryIdType'],
+            failure_reason: response_as_json['failureReason'],
+            transaction_end_date_time: response_as_json['transactionEndDateTime'],
+            transaction_start_date_time: response_as_json['transactionStartDateTime'],
+            status: response_as_json['status'],
+            assurance_level: response_as_json['assuranceLevel'],
+            proofing_post_office: response_as_json['proofingPostOffice'],
+            proofing_city: response_as_json['proofingCity'],
+            proofing_state: response_as_json['proofingState'],
+            scan_count: response_as_json['scanCount'],
+            response_message: response_as_json['responseMessage'],
```

Comment:
Could we move this all to the 'enrollment with a status update' shared example by doing this?

```
        before do
          stub_request_passed_proofing_results
        end

        it_behaves_like(
          'enrollment with a status update',
          passed: true,
          response_json: UspsInPersonProofing::Mock::Fixtures.request_passed_proofing_results_response,
          status: 'passed',
        )
```

And then in the shared example we could do this:

```
RSpec.shared_examples 'enrollment with a status update' do |passed:, status:, response_json:|
  it 'logs a message with common attributes' do
    <...omitted code...>
    response = JSON.parse(response_json)
    expect(job_analytics).to have_logged_event(
      'GetUspsProofingResultsJob: Enrollment status updated',
      assurance_level: response['assuranceLevel'],
      enrollment_code: pending_enrollment.enrollment_code,
      enrollment_id: pending_enrollment.id,
      failure_reason: response['failureReason'],
      fraud_suspected: response['fraudSuspected'],
      minutes_since_last_status_check: 15.0,
      minutes_since_last_status_update: 2.days.in_minutes,
      minutes_to_completion: 3.days.in_minutes,
      passed: passed,
      primary_id_type: response['primaryIdType'],
      proofing_city: response['proofingCity'],
      proofing_post_office: response['proofingPostOffice'],
      proofing_state: response['proofingState'],
      response_message: response['responseMessage'],
      scan_count: response['scanCount'],
      secondary_id_type: response['secondaryIdType'],
      status: response['status'],
      transaction_end_date_time: response['transactionEndDateTime'],
      transaction_start_date_time: response['transactionStartDateTime'],
    )
  end
```

And we could then use that shared example for most the test cases where you've added these checks. I like this approach because the shared examples already check for logged attributes and this would reduce a lot of similar looking lines of test code. And then we could revert the changes to usps_ipp_helper.rb